scavenger-api/src/main/kotlin/com/navercorp/scavenger/service/ExportSnapshotMethodService.kt

```diff
@@ -0,0 +1,65 @@
+package com.navercorp.scavenger.service
+
+import com.github.doyaaaaaken.kotlincsv.dsl.csvWriter
+import com.navercorp.scavenger.dto.ExportSnapshotMethodDto
+import com.navercorp.scavenger.entity.ExportSnapshotMethodEntity
+import com.navercorp.scavenger.repository.ExportSnapshotMethodRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.io.IOException
+import java.io.OutputStream
+
+@Service
+class ExportSnapshotMethodService(
+    var exportSnapshotMethodRepository: ExportSnapshotMethodRepository
+) {
+
+    @Transactional(readOnly = true)
+    fun writeDtoToTsv(stream: OutputStream, customerId: Long, snapshotId: Long) {
+        val data: List<ExportSnapshotMethodEntity> = exportSnapshotMethodRepository.findSnapshotMethodExport(customerId, snapshotId)
+
+        try {
+            val rows = mutableListOf(
+                listOf(
+                    "filterInvokedAtMillis",
+                    "packages",
+                    "status",
+                    "excludeAbstract",
+                    "parent",
+                    "signature",
+                    "type",
+                    "usedCount",
+                    "unusedCount",
+                    "lastInvokedAtMillis"
+                )
+            )
+
+            data.forEach { entity: ExportSnapshotMethodEntity ->
+                val dto = ExportSnapshotMethodDto.from(entity)
+                rows.add(
+                    listOf(
```

Comment:
How about creating a `toCsv()` in `ExportSnapshotMethodDto` and using it?

Reply:
This makes for more readable code, as shown below.
```kotlin
            val rows = listOf(
                listOf(
                    "filterInvokedAtMillis",
                    "packages",
                    "status",
                    "excludeAbstract",
                    "parent",
                    "signature",
                    "type",
                    "usedCount",
                    "unusedCount",
                    "lastInvokedAtMillis"
                )
            ) +
                exportSnapshotMethodRepository.findSnapshotMethodExport(customerId, snapshotId)
                    .map { ExportSnapshotMethodDto.from(it).toCsv() }
```

Reply:
I fixed this review on commit 5b18e46.
The modifications in this commit are slightly different from what you mentioned. I tried to implement it as close as possible using the kotlin-csv library you recommended.